@types/pal/input.d.ts

```diff
@@ -73,4 +202,173 @@ declare module 'pal/input' {
          */
         public on (eventType: import('cocos/input/types/event-enum').InputEventType, callback: AccelerometerCallback, target?: any);
     }
+
+    type HandleCallback = (res: import('cocos/input/types').EventHandle) => void;
+    /**
+     * Class designed for handle input.
+     */
+    export class HandleInputDevice {
```

Comment:
HandleInput or HandInput ?

Reply:
Not hand gesture, just handle devices input. There would be another HandInput in the future : )